scripts/data.js

```diff
@@ -0,0 +1,32 @@
+const fs = require("fs");
```

Comment:
Looks like `data.js` and `package.js` do almost the same, is it ok?

Reply:
`scripts` folder is in `.npmignore`. So it's for us, in case of changes.